shared/chat/conversations-list/container.js

```diff
@@ -109,8 +112,10 @@ function _filterInboxes (inboxes: List<InboxState>, supersededByState: Supersede
 export default connect(
   (state: TypedState, {routeSelected}) => ({
     conversationUnreadCounts: state.chat.get('conversationUnreadCounts'),
-    inbox: _filterInboxes(state.chat.get('inbox'), state.chat.get('supersededByState'), state.chat.get('alwaysShow')),
```

Comment:
don't run this filter unless something has actually changed. use connect() internal optimization for this